src/auth/auth.controller.ts

```diff
@@ -38,4 +46,25 @@ export class AuthController {
   async register(@Body() userDto: CreateUserDto) {
     return this.authService.register(userDto);
   }
+
+  @Post("login")
+  @UseGuards(LocalAuthGuard)
+  @ApiOperation({
+    description:
+      "email or account as login account.  \n" +
+      "When the account information is correct will return token.",
+    summary: "user local login",
+  })
+  @ApiCreatedResponse({
+    description: "Generate token",
+    type: GenerateTokenRespose,
+  })
+  @ApiUnauthorizedResponse({
+    description: "Account information error",
+    type: SelectUnauthorizedError,
+  })
```

Comment:
as described, I would use 403 instead of 401.

Reply:
The problem is same.
https://github.com/Cophr/backend/pull/10#discussion_r1257520394